### DISCUSSION TITLE
resume

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#693] [Announcements]
# resume

个人信息 ::: tip 基本信息
倪传方/男/1994.08.20; 
本科/合肥工业大学-工程力学; 
工作年限: 4 年; 
工作单位: 鸿程计算机系统有限公司-至今; 
Github: ; 
Gitee: ; 
期望职位: Java 开发工程师,python 开发工程师; 
当前薪资: 11k; 
期望薪资: 税前月薪 15k+; 
期望城市：...

http://localhost:8080/resume

**Comment by nichuanfang** [comment]
test
